torch/csrc/jit/tracer.cpp

```diff
@@ -39,6 +39,26 @@ thread_local std::shared_ptr<TracingState> tracing_state;
 
 } // namespace detail
 
+// Hack to emulate a lambda with move capture on non-c++11 targets
```

Comment:
Can you just use the clang tidy off pragma instead? This is a lot of cruft just to appease a stupid rule.